src/components/Input/Input.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './Input.module.scss';
+import PropTypes from 'prop-types';
+
+const Input = ({ tag: Tag, name, type, label, maxLength, ...props }) => (
+	<div className={styles.inputWrapper}>
+		<input
+			className={Tag !== 'searchInput' ? styles.input : styles.searchInput}
```

Comment:
Instead of hardcoded strings, use what you did for the `FormTypes` and use "enums"